{
  "deprecated": false,
  "cpeName": "cpe:2.3:a:ibm:inotes:9.0.1.8:*:*:*:*:*:*:*",
  "cpeNameId": "3B2FC929-6F61-4035-A14C-2874682C8565",
  "created": "2019-10-29T16:21:19.930",
  "lastModified": "2019-10-29T16:21:19.930",
  "titles": [
    {
      "title": "IBM iNotes 9.0.1.8",
      "lang": "en"
    }
  ],
  "refs": [
    {
      "ref": "https://exchange.xforce.ibmcloud.com/vulnerabilities/126234",
      "type": "Version"
    },
    {
      "ref": "https://www.ibm.com/us-en/",
      "type": "Vendor"
    },
    {
      "ref": "https://www-945.ibm.com/support/fixcentral/options?selectionBean.selectedTab=find&selection=Collaboration+Solutions%3bibm%2fLotus%2fLotus+iNotes",
      "type": "Version"
    }
  ]
}
